### DISCUSSION TITLE
Stickers

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#4075] [Polls]
If we made a (very cool) Outline sticker pack would that be of interest? Thinking it could potentially make a nice sponsor reward…

**Comment by furilo** [comment]
I'd buy lots of stickers if they come with the comments feature ;)

**Reply by tommoor** [reply]
More stickers = More sponsorship = features getting built faster 😉

**Comment by Limezy** [comment]
Do you ship worldwide 🌎?!
